test/UserService.Business.UnitTests/EditUserCommandTests.cs

```diff
@@ -162,6 +162,8 @@ public void ShouldEditUserWhenUserDataIsValidAndCurrentUserHasRights()
                 .Setup(x => x.IsAdmin(null))
                 .Returns(false);
 
+            ClientRequestUp(Guid.NewGuid());
```

Comment:
for what this line in test?

Reply:
That set new Id to httpContext and it will  != userId

Reply:
Emulating that another user trying to change user